app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -252,13 +255,18 @@
     private PlayQueueAdapter playQueueAdapter;
     private StreamSegmentAdapter segmentAdapter;
 
-    @Nullable private MediaSourceManager playQueueManager;
+    @Nullable
+    private MediaSourceManager playQueueManager;
```

Comment:
Please do not reformat these

Reply:
If these shouldn't be formatted, maybe a
```
// @formatter:off
...
// @formatter:on 
```
block would be a good idea:
https://stackoverflow.com/questions/3375307/how-to-disable-code-formatting-for-some-part-of-the-code-using-comments

Is there anywhere a formatter configuration file that I failed to import?

Reply:
My Android Studio does nothing with those lines, maybe you have some particular settings?

Reply:
@Stypox 
The option "Do not warp after single annotation" is not enabled in my Android Studio:
![androidStudioSettings](https://user-images.githubusercontent.com/40789489/121410790-40245400-c963-11eb-82b8-0512ae703e6d.png)

If I reset my Java Code Style Settings (Settings>Editor>Code Style>Java) the option is also disabled and also when I import the checkstyle config, which results in the reformatted code above.

Reply:
Mmmh, I wonder how that can be disabled automatically for whoever imports the NewPipe project (I have no idea how config files work)

Reply:
I can recommend exporting settings as ``.editorconfig`` (although I never did it in IntelliJ - just in Visual Studio and VS Code)
![grafik](https://user-images.githubusercontent.com/40789489/121412851-69de7a80-c965-11eb-937d-cb00c1c71679.png)

see also: https://editorconfig.org/

Another great thing that should be put onto a setup guide or something similar (which I still miss as a newcomer) → https://github.com/TeamNewPipe/NewPipe/issues/6220